### ISSUE TITLE
Loss not changing when training 

### ISSUE THREAD

**Issue by RenukaNale** [tensorflow/models#2814]
I am using tensorflow object detection api for my own dataset I am facing some problem.  I am using centos , with GPU Geforce 1080, 8 GB GPU memory, tensorflow 1.2.1 . I did the following steps and I  have two problems.
1. I annotated my images using LabelImg tool
2. Created tfrecord successfully
3.I used ssd_inception_v2_coco.config. I modified the only path, no of class and I did not train from scratch, I used ssd_inception_v2_coco model checkpoints.
Problem 1: from step 0 until 3000, my loss has dramatically decreased but after that, it stays constant between 5 to 6 . Not getting how I reduce it but still my model able to detect required object. 
Even i tried for diffent model eg. faster_rcnn_inception_resnet_v2_atrous_coco after some steps loss stay constant between 1 and 2
Problem 2:   according to a document I able to run eval.py  but getting the following error:
WARNING:root:The following classes have no ground truth examples: 0 after that program terminate.
I try to run train.py and eval.py at the same time still same error.
Please give me a suggestion. I am tensorflow beginner sorry for stupid question.

**Comment by jmayer13** [comment]
I belief that the "default" model is broken #2812
Could you try another model like *faster_rcnn_resnet101_coco_2017_11_08* to see if something changes?

**Comment by RenukaNale** [comment]
Thank you for the suggestion but I am not getting what do you mean that the "default" model is broken.

**Comment by aselle** [comment]
Change the model name to "faster_rcnn_resnet101_coco_2017_11_08"  as 
specified here ... https://github.com/tensorflow/models/issues/2773

**Comment by RenukaNale** [comment]
Thank you for a suggestion I will try faster_rcnn_resnet101_coco_2017_11_08. 
I only tried faster_rcnn_inception_resnet_v2_atrous_coco-from step 0 until 3000, my loss has dramatically decreased but after that, it stays constant between 1 to 2.
ssd_inception_v2_coco and ssd_mobilenet_v1_coco- same from step 0 until 3000, my loss has dramatically decreased but after that, it stays constant between 5 to 6,

**Comment by drpngx** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by Shruthi-Sampathkumar** [comment]
Did you convert the labels to one hot encoded form while conversion to tfrecord?@RenukaNale
